stdlib/Profile/src/Profile.jl

```diff
@@ -299,104 +318,113 @@ error_codes = Dict(
 """
     fetch() -> data
 
-Returns a reference to the internal buffer of backtraces. Note that subsequent operations,
-like [`clear`](@ref), can affect `data` unless you first make a copy. Note that the
+Returns a copy of the buffer of profile backtraces. Note that the
```

Comment:
I don't remember why it was this way before, but this seems like a good change.

Reply:
I assume it was to be zero-copy, but it's just not that useful given all the other work this needs to do, and simpler to be able to pre-process the buffer here.